code/modules/modular_computers/file_system/programs/ntmonitor.dm

```diff
@@ -56,12 +56,23 @@
 			return 1
 		if("purgelogs")
 			. = 1
+			var/mob/user = usr
+			if(computer.computer_emagged == 0)
```

Comment:
Noo. Leave this. I ALREADY put hyper security into the program by making you need access to run it rather than just download.